### ISSUE TITLE
DatasetSelectorDialog starting API

### ISSUE THREAD

**Issue by xulman** [saalfeldlab/n5-ij#95]
Minimal opening example

```
final String zarrRootStrPath = "/temp/Zurich.hackathon.testData/maybe_top_level.zarr";
DatasetSelectorDialog selectionDialog = new DatasetSelectorDialog(
				  new N5Importer.N5ViewerReaderFun(),
				  new N5Importer.N5BasePathFun(),
				  zarrRootStrPath,
				  new N5MetadataParser[]{new OmeNgffMetadataParser()},
				  PARSERS);

//just an empty function, otherwise NPE is thrown
selectionDialog.setContainerPathUpdateCallback(s -> {});

selectionDialog.run(ds -> {
     System.out.println("just some data is selected...");
});

//makes the dialog to update (and thus display) its content
selectionDialog.detectDatasets();
```

When working with the pointed at file/path, the dialog is calling its `containerPathUpdateCallback`, which is normally initiated as `null`.

So, one has to provide there simply anything... to avoid having it getting NPE.
And the **issue** is that it is not obvious and perhaps the empty callback could be a better initial setting than just null.
What do you think? @bogovicj

**Comment by xulman** [comment]
I have of course forgotten to suggest what to do with it:

replace [current code](https://github.com/saalfeldlab/n5-ij/blob/84c945f1396976719608d6acb16ef2160b80578a/src/main/java/org/janelia/saalfeldlab/n5/ui/DatasetSelectorDialog.java#L156)
with
```
private Consumer<String> containerPathUpdateCallback = s -> {};
```

**Comment by bogovicj** [comment]
@xulman 
I agree, if you're interested in doing this, I would happily accept a PR, otherwise I'll get to it eventually.

**Comment by xulman** [comment]
actually, the PR #96 is solving this issue  (by getting access to the original code base rather than trying to construct and use the dialog itself directly)

**Comment by xulman** [comment]
Since the PR #96 is merged now, this issue is solved.
